src/IssueViz.Security/Hotspots/HotspotReviewPriorityProvider.cs

```diff
@@ -0,0 +1,222 @@
+            { "typescript:S2612", HotspotPriority.Medium },
+            { "typescript:S5443", HotspotPriority.Low },
+            { "typescript:S5689", HotspotPriority.Low },
+            { "typescript:S6319", HotspotPriority.Low },
+            { "typescript:S4818", HotspotPriority.Low },
+            { "typescript:S4817", HotspotPriority.Low },
+            { "typescript:S2077", HotspotPriority.High },
+            { "typescript:S5693", HotspotPriority.Medium },
+            { "typescript:S5691", HotspotPriority.Low },
+            { "typescript:S6308", HotspotPriority.Low },
+            { "typescript:S6302", HotspotPriority.Medium },
+            { "typescript:S6303", HotspotPriority.Low },
+            { "typescript:S2068", HotspotPriority.High },
+            { "typescript:S5332", HotspotPriority.Low },
+            { "typescript:S6304", HotspotPriority.Medium },
+            { "typescript:S6299", HotspotPriority.High },
+            { "typescript:S4790", HotspotPriority.Low },
+            { "typescript:S2255", HotspotPriority.Low },
+            { "typescript:S6281", HotspotPriority.Medium },
+            { "typescript:S5759", HotspotPriority.Low },
+            { "typescript:S4784", HotspotPriority.Low },
+            { "typescript:S3330", HotspotPriority.Low },
+            { "typescript:S5757", HotspotPriority.Low },
+            { "typescript:S2245", HotspotPriority.Medium },
+            { "typescript:S4787", HotspotPriority.Low },
+            { "typescript:S6252", HotspotPriority.Low },
+            { "typescript:S5042", HotspotPriority.Low },
+            { "typescript:S5728", HotspotPriority.Low },
+            { "typescript:S5725", HotspotPriority.Low },
+            { "typescript:S5604", HotspotPriority.Low },
+            { "typescript:S4507", HotspotPriority.Low },
+            { "typescript:S6245", HotspotPriority.Low },
+            { "typescript:S4502", HotspotPriority.High },
+            { "typescript:S6249", HotspotPriority.Low },
+            { "typescript:S6270", HotspotPriority.Medium },
+            { "typescript:S6275", HotspotPriority.Low },
+            { "typescript:S5742", HotspotPriority.Low },
+            { "typescript:S5743", HotspotPriority.Low },
+            { "typescript:S6265", HotspotPriority.Medium },
+            { "typescript:S5739", HotspotPriority.Low },
+            { "typescript:S5736", HotspotPriority.Low },
+            { "typescript:S5730", HotspotPriority.Low },
+            { "typescript:S5852", HotspotPriority.Medium },
+            { "typescript:S6268", HotspotPriority.High },
+            { "typescript:S5734", HotspotPriority.Low },
+            { "typescript:S5732", HotspotPriority.Low },
+            { "typescript:S6350", HotspotPriority.High },
+        };
+
+        public HotspotPriority? GetPriority(string hotspotKey)
+        {
+            return hotspotKey == null || !KeyToPriorityMap.TryGetValue(hotspotKey, out var priority) ? (HotspotPriority?)null : priority;
```

Comment:
Calling this method with `null` would be an error in our code, wouldn't it?

I wouldn't code defensively in this case; I'd throw an exception.

Reply:
Added null check